net-passwd.h-example

```diff
@@ -9,6 +9,8 @@
 
 // IP Address of the MQTT BROCKER
 #define myMQTT_BROCKER "192.168.1.1"
+#define myMQTT_USERNAME "user"
+#define myMQTT_PASSWORD "pass"
```

Comment:
Better: comment this out in the example file and provide a default configuration in MQTT433gateway.ino.